src/scripts/main.js

```diff
@@ -1,5 +1,28 @@
 'use strict';
 
 document.addEventListener('click', (e) => {
-  // write code here
+  if (e.target.className === 'wall') {
```

Comment:
You're using a strict equality check against a string literal here. This can be error-prone if the class name changes or if more classes are added to the element. A safer approach would be to use the `classList.contains` method to check if the element has the specific class.